CQDOC-20114 updating msft scopes

abhishekgarg18 opened the pull request.

alexkli commented:
> Adobe CLA Signed?

I am an Adobe employee, so something is wrong with this check (it works on other OS repos for me).

anujkapo commented:
Thanks, @alexkli for reporting the issue. Tracked using CQDOC-17186.

aheim0 commented:
Thanks for highlighting this, the updates have been applied.
Closing issue.

aheim0 committed:
Merge branch 'main' into main

aheim0 committed:
Merge pull request #3434 from plegault/main

Additional impl not needed

aheim0 committed:
Merge branch 'main' into CFGQL-PSHF-01

aheim0 committed:
Merge branch 'main' into patch-2

aheim0 committed:
Merge pull request #3407 from pasahu/patch-2

Updated the yaml to avoid authentication error & making it generic

aheim0 committed:
Merge branch 'main' into feature/CIF-AEP-Integration

aheim0 committed:
Merge branch 'main' into cf2_screenshot_update

aheim0 committed:
Merge branch 'main' into issue/CIF-2720

aheim0 committed:
Merge pull request #3116 from drudolph/issue/CIF-2720

CIF-2720: update specific page documentation for multiple catalog pages

aheim0 committed:
Merge branch 'main' into mnew-patch-1

aheim0 committed:
Merge branch 'main' into mnew-UserMapping-and-emails

aheim0 committed:
Merge branch 'main' into mnew-UserMapping-and-emails-1

aheim0 committed:
Merge branch 'main' into mnew-doc-UMPermissions

aheim0 committed:
Merge pull request #2752 from AdobeDocs/mnew-doc-UMPermissions

Update using-user-mapping-tool.md

raimanS committed:
Merge branch 'main' into cf2_screenshot_update

raimanS committed:
Merge pull request #3240 from saluja/cf2_screenshot_update

Update cloudfront2.png

atopper committed:
@trivial Add reminder to submit ingestion support ticket

atopper committed:
@trivial Add reminder to submit ingestion support ticket

aheim0 committed:
Merge branch 'main' into CFGQL-PSHF-01

raimanS committed:
update 1

raimanS committed:
Merge pull request #3447 from AdobeDocs/500-error

update 1

raimanS committed:
update 2

raimanS committed:
Merge branch 'main' into 500-error

raimanS committed:
update 3

raimanS committed:
Merge branch '500-error' of https://git.corp.adobe.com/AdobeDocs/experience-manager-cloud-service.en into 500-error

raimanS committed:
Merge pull request #3448 from AdobeDocs/500-error

update 2

raimanS committed:
update 5

raimanS committed:
Merge branch 'main' into 500-error

raimanS committed:
Merge pull request #3449 from AdobeDocs/500-error

update 5

sarchiz committed:
Merge branch 'main' into mnew-patch-1

sarchiz committed:
Merge pull request #2939 from AdobeDocs/mnew-patch-1

Update overview-user-mapping-tool.md

TylerRush committed:
Update screens.md

sarchiz committed:
Merge branch 'main' into mnew-UserMapping-and-emails

TylerRush committed:
Merge branch 'main' into TRNov23

sarchiz committed:
Merge pull request #2926 from AdobeDocs/mnew-UserMapping-and-emails

Update considerations-user-mapping-tool.md

sarchiz committed:
Merge branch 'main' into mnew-UserMapping-and-emails-1

TylerRush committed:
Update screens.md

TylerRush committed:
Merge branch 'TRNov23' of https://git.corp.adobe.com/AdobeDocs/experience-manager-cloud-service.en into TRNov23

TylerRush committed:
Merge branch 'main' into TRNov23

sarchiz committed:
Merge pull request #2925 from AdobeDocs/mnew-UserMapping-and-emails-1

Update using-user-mapping-tool.md

TylerRush committed:
Merge branch 'main' into TRNov23

TylerRush committed:
Merge pull request #3450 from AdobeDocs/TRNov23

Update screens.md

guillaumecarlino committed:
Merge branch 'main' into feature/CIF-AEP-Integration

bohnertchris committed:
Rewrite Content Fragment model

bohnertchris committed:
Rearrange content and number steps

bohnertchris committed:
Consistent capitalization

bohnertchris committed:
Rewrite create Content Fragment

bohnertchris committed:
Rewrite extracting content

bohnertchris committed:
Unify structure and cross link

bohnertchris committed:
Update image links

bohnertchris committed:
More cross-linking

bohnertchris committed:
Rewrite first few steps of customize-app

bohnertchris committed:
Finish rewrite of customize-app

atopper committed:
@trivial Add reminder to submit ingestion support ticket
- extra spaces around 'top-up ingestion' might be confusing translations

atopper committed:
@trivial Add reminder to submit ingestion support ticket
- not putting title & abstract on different lines may confuse translations

aheim0 committed:
Merge branch 'main' into CFGQL-PSHF-01

bohnertchris committed:
Redact beta/staging info from images

bohnertchris committed:
Proofreading

bohnertchris committed:
Applying Acrolinx recommendations that actually make sense

bohnertchris committed:
Merge branch 'main' into aem-trials

kkour06 committed:
modified

bohnertchris committed:
Fixed broken image

kkour06 committed:
minor modifications

raimanS committed:
update 1

raimanS committed:
Merge pull request #3456 from AdobeDocs/note-update

update 1

kkour06 committed:
Minor changes

kkour06 committed:
minor changes

sarchiz committed:
Merge branch 'main' into GRANITE-41654

bohnertchris committed:
Update help/implementing/developing/introduction/development-guidelines.md

Co-authored-by: Marcel Reutegger <mreutegg@adobe.com>

bohnertchris committed:
Merge branch 'main' into CQDOC-19781

bohnertchris committed:
Merge pull request #3403 from AdobeDocs/CQDOC-19781

CQDOC-19781

sarchiz committed:
Merge branch 'main' into GRANITE-41654

guillaumecarlino committed:
Update indexing.md

guillaumecarlino committed:
Merge branch 'main' into feature/CIF-AEP-Integration

sarchiz committed:
Merge branch 'main' into top/ro_reminder

sarchiz committed:
Merge pull request #3446 from AdobeDocs/top/ro_reminder

@trivial Add reminder to submit ingestion support ticket

bohnertchris committed:
Feedback from Alison

bohnertchris committed:
Another catch from Alison

rickbrough committed:
CQDOC-19737

bohnertchris committed:
Merge branch 'main' into aem-trials

bohnertchris committed:
Updated previews of apps

rickbrough committed:
Merge pull request #3457 from AdobeDocs/rb112822

CQDOC-19737

aheim0 committed:
Merge branch 'main' into CFGQL-PSHF-01

bohnertchris committed:
Archive old page and add to ToC

bohnertchris committed:
Copy-paste release notes

bohnertchris committed:
Make release notes readable

sarchiz committed:
attempt to fix validation

sarchiz committed:
Merge branch 'main' into GRANITE-41654

sarchiz committed:
Merge pull request #3354 from nitigup/GRANITE-41654

GRANITE-41654 | [Documentation] Skyline index creation with newest ve…

anujkapo committed:
Added the full document preview support for MSFT files

anujkapo committed:
Merge pull request #3459 from AdobeDocs/file-format-support

Added the full document preview support for MSFT files

bohnertchris committed:
Forgot to rename the old release notes file

bohnertchris committed:
Merge branch 'main' into cloud-manager-2022.12.0

bohnertchris committed:
Merge pull request #3458 from AdobeDocs/cloud-manager-2022.12.0

Cloud manager 2022.12.0

bohnertchris committed:
Merge branch 'main' into aem-trials

guillaumecarlino committed:
Merge branch 'main' into feature/CIF-AEP-Integration

guillaumecarlino committed:
Update release-notes-current.md

guillaumecarlino committed:
Merge pull request #3370 from AdobeDocs/feature/CIF-AEP-Integration

Feature/cif aep integration

Commit:
[[DNT - DO NOT TRANSLATE]] Adding ExL tracking IDs to meta data

aheim0 committed:
Merge branch 'main' into CFGQL-PSHF-01

bohnertchris committed:
Clarification of delimiting fields

bohnertchris committed:
Merge branch 'main' into aem-trials

TylerRush committed:
Update replication.md

Added the note requested from CQDOC-19839

TylerRush committed:
Merge branch 'main' into TRNov30

bohnertchris committed:
Merge pull request #3355 from AdobeDocs/aem-trials

AEM Trials documentation

TylerRush committed:
Merge branch 'main' into TRNov30

TylerRush committed:
Update replication.md

TylerRush committed:
Merge branch 'TRNov30' of https://git.corp.adobe.com/AdobeDocs/experience-manager-cloud-service.en into TRNov30

TylerRush committed:
Merge pull request #3460 from AdobeDocs/TRNov30

Update replication.md

froesef committed:
Add example customer files to file structure as clarification

TylerRush committed:
Update maintenance.md

Updates for CQDOC-19863

TylerRush committed:
Merge branch 'main' into TRNov30

TylerRush committed:
Merge pull request #3462 from AdobeDocs/TRNov30

Update maintenance.md

davidjgonzalez committed:
Add note about configuring the Referrer Filter.

Configuring the Referrer Filter may break AEM's Content Fragment console and Assets Essentials functionality, as the custom Referrer Filter configuration will replace the OOTB configuration relied on by AEM.

TylerRush committed:
Update caching.md

TylerRush committed:
Merge branch 'main' into TRNov30

TylerRush committed:
Merge pull request #3464 from AdobeDocs/TRNov30

Update caching.md

rickbrough committed:
Added "Guidelines for defining Smart Crop in an Image profile"

See under "Crop options" heading.

rickbrough committed:
Merge branch 'main' into rb113022

rickbrough committed:
Merge pull request #3465 from AdobeDocs/rb113022

Added "Guidelines for defining Smart Crop in an Image profile"

Commit:
[[DNT - DO NOT TRANSLATE]] Adding ExL tracking IDs to meta data

aheim0 committed:
Merge branch 'main' into CFGQL-PSHF-01

kkour06 committed:
Merge pull request #3455 from AdobeDocs/CQDOC-19147-k

modified

anujkapo committed:
Incorporating PM feedback

anujkapo committed:
Merge branch 'main' into workfront-integrations-new

anujkapo committed:
Merge pull request #3466 from AdobeDocs/workfront-integrations-new

Incorporating PM feedback

rickbrough committed:
Update image-profiles.md

rickbrough committed:
Merge branch 'main' into rb113022

rickbrough committed:
Merge pull request #3467 from AdobeDocs/rb113022

Update image-profiles.md

FrancoisZhang committed:
CQDOC-19881 fix wrong documentation for asset name sanitization of bulk import

Ruchita454401 committed:
Update generate-document-of-record-for-non-xfa-based-adaptive-forms.md

Ruchita454401 committed:
DoR Update

Ruchita454401 committed:
Merge pull request #3469 from AdobeDocs/dorupdate/ruchita

Update generate-document-of-record-for-non-xfa-based-adaptive-forms.md

aheim0 committed:
Merge branch 'main' into CFGQL-PSHF-01

anujkapo committed:
Merge branch 'main' into CQDOC-19881

anujkapo committed:
Merge pull request #3468 from zjun/CQDOC-19881

CQDOC-19881 fix wrong documentation for asset name sanitization of bu…

kkour06 committed:
Merge branch 'main' into CQDOC-19741-k

kkour06 committed:
Merge pull request #3377 from AdobeDocs/CQDOC-19741-k

region and egress IPs added

kkour06 committed:
Merge branch 'main' into CQDOC-19638-k

kkour06 committed:
Merge pull request #3324 from AdobeDocs/CQDOC-19638-k

updated as per requirement

rickbrough committed:
Update dm-journey-part1.md

Added {target="_blank"} at end of url link to open link into new tab.

rickbrough committed:
Merge pull request #3470 from AdobeDocs/rb120522

Update dm-journey-part1.md

rickbrough committed:
Update limitations.md

rickbrough committed:
Merge branch 'main' into rb120522

rickbrough committed:
Added syntax to open new tab in "Try it" parts.

rickbrough committed:
Merge branch 'rb120522' of https://git.corp.adobe.com/AdobeDocs/experience-manager-cloud-service.en into rb120522

rickbrough committed:
Merge pull request #3471 from AdobeDocs/rb120522

Update limitations.md

rickbrough committed:
Update dm-journey-part2.md

rickbrough committed:
Merge branch 'main' into rb120522

rickbrough committed:
Merge pull request #3472 from AdobeDocs/rb120522

Update dm-journey-part2.md

aheim0 committed:
CFGQL-PSHF-01-04

aheim0 committed:
Merge branch 'main' into CFGQL-PSHF-01

sarchiz committed:
Update maintenance.md

sarchiz committed:
Merge pull request #3473 from AdobeDocs/sapatch-6dec

Update maintenance.md

aheim0 committed:
XF-HTMLRG-01-01

aheim0 committed:
Merge pull request #3474 from AdobeDocs/XF-HTMLRG-01

XF-HTMLRG-01-01

aheim0 committed:
Merge branch 'main' into CFGQL-PSHF-01

aheim0 committed:
Merge branch 'main' into dgonzale-patch-1

aheim0 committed:
Merge pull request #3463 from AdobeDocs/dgonzale-patch-1

Add note about configuring the Referrer Filter.

raimanS committed:
Merge branch 'main' into dispatcher-clarification

raimanS committed:
Merge pull request #3461 from ffroese/dispatcher-clarification

Add example customer files to file structure as clarification

bohnertchris committed:
Added enhanced security option for HIPAA

bohnertchris committed:
Pointer to docs on production programs

bohnertchris committed:
Initial cleanup of existing page content

atopper committed:
CQ-4350533 [CAM] Update documentation to explain 'ro running'

bohnertchris committed:
Editing enhanced security program

bohnertchris committed:
Merge pull request #3475 from AdobeDocs/CQDOC-19798

CQDOC-19798

BlakeFrei committed:
Updates to metadata and landings for SEO

BlakeFrei committed:
Merge pull request #3479 from AdobeDocs/seo-updates-landings

Updates to metadata and landings for SEO

aheim0 committed:
Merge branch 'main' into CFGQL-PSHF-01

aheim0 committed:
CQGQL-PSHF-01-PR3476

aheim0 committed:
CFGQL-PSHG-01-PR3441

aheim0 committed:
CFGQL-PSHF-01-A01

aheim0 committed:
Merge pull request #3411 from AdobeDocs/CFGQL-PSHF-01

CFGQL-PSHF-01-01

sarchiz committed:
Merge branch 'main' into top/CQ-4350533-ro-running

sarchiz committed:
Merge pull request #3478 from AdobeDocs/top/CQ-4350533-ro-running

CQ-4350533 [CAM] Update documentation to explain 'ro running'

bohnertchris committed:
Integrated video content into doc

bohnertchris committed:
Merge branch 'main' into CQDOC-19914

bohnertchris committed:
Typo corrections

bohnertchris committed:
Remove low-quality video

bohnertchris committed:
Merge pull request #3477 from AdobeDocs/CQDOC-19914

CQDOC-19914

anujkapo committed:
Workfront enhanced connector release 1.9.6

anujkapo committed:
Merge branch 'main' into workfront-enhanced-connector-196

anujkapo committed:
PM feedback

anujkapo committed:
Merge branch 'workfront-enhanced-connector-196' of https://git.corp.adobe.com/AdobeDocs/experience-manager-cloud-service.en into workfront-enhanced-connector-196

anujkapo committed:
PM comments

anujkapo committed:
Minor fix

anujkapo committed:
Minor addition

TylerRush committed:
Update user-access-new-relic.md

Removed reference to Access Managed by Adobe support as requested by email from Mathew Laun on 12/09/22

TylerRush committed:
Merge branch 'main' into TRDec9

anujkapo committed:
Minor additions

TylerRush committed:
Merge pull request #3482 from AdobeDocs/TRDec9

Update user-access-new-relic.md

anujkapo committed:
Merge branch 'main' into workfront-enhanced-connector-196

rickbrough committed:
Merge pull request #3481 from AdobeDocs/workfront-enhanced-connector-196

Workfront enhanced connector release 1.9.6

aheim0 committed:
CFGQL-FU-202212-01-01

aheim0 committed:
Merge pull request #3486 from AdobeDocs/CFGQL-FU-202212-01

CFGQL-FU-202212-01-01

aheim0 committed:
CFGQL-FU-202212-02-01

aheim0 committed:
Merge pull request #3493 from AdobeDocs/CFGQL-FU-202212-02

CFGQL-FU-202212-02-01

aheim0 committed:
CFGQL-FU-202212-03-01

aheim0 committed:
Merge pull request #3494 from AdobeDocs/CFGQL-FU-202212-03

CFGQL-FU-202212-03-01

guillaumecarlino committed:
CQDOC - 19923 (#3491)

Co-authored-by: David Gonzalez <dgonzale@adobe.com>

aheim0 committed:
GH40-DISO-01-01

aheim0 committed:
Merge pull request #3495 from AdobeDocs/GH40-DISO-01

GH40-DISO-01-01

aheim0 committed:
GH380-TTBD-01-01

aheim0 committed:
Merge pull request #3496 from AdobeDocs/GH380-TTBD-01

GH380-TTBD-01-01

raimanS committed:
update 1

raimanS committed:
Merge pull request #3497 from AdobeDocs/note-update

update 1

bohnertchris committed:
Added feedback from gwalt

bohnertchris committed:
Merge pull request #3499 from AdobeDocs/CQDOC-18208

CQDOC-18208

bohnertchris committed:
Feedback from gwalt

bohnertchris committed:
Duplicate headings

bohnertchris committed:
Merge pull request #3500 from AdobeDocs/CQDOC-18313

CQDOC-18131

aditidc committed:
Update release-notes-migration-tools-current.md

aditidc committed:
Create release-notes-migration-tools-2022-09-0.md

sarchiz committed:
add TOC entry for september release notes

sarchiz committed:
remove duplicate exl-id

sarchiz committed:
Merge pull request #3503 from AdobeDocs/duttchau-patch-3

Update release-notes-migration-tools-current.md

singhkh committed:
Forms CS PDF Utility (#3489)

bohnertchris committed:
Clarified that the app should be installed as a package

bohnertchris committed:
Update with WKND sample

bohnertchris committed:
Added warnings back to content

bohnertchris committed:
Merge pull request #3506 from AdobeDocs/CQDOC-18603

CQDOC-18603

noozoo committed:
Correction on user mapping

Groups are not mapped, so removed that.

bohnertchris committed:
Typos

bohnertchris committed:
Adding info about FE contract

sarchiz committed:
Merge pull request #3512 from AdobeDocs/mnew-patch-2

Correction on user mapping

bohnertchris committed:
Clarification

bohnertchris committed:
Cross-linked the documentation

bohnertchris committed:
Proper spacing

bohnertchris committed:
Merge branch 'main' into CQDOC-18662

bohnertchris committed:
Merge pull request #3513 from AdobeDocs/CQDOC-18662

CQDOC-18662

atopper committed:
CQ-4350666 [CAM] Update documentation to explain oak uuid conflicts

ycastane committed:
Update user-access-new-relic.md

Updated the number of users allowed in NR APM offering.  Two sections have different numbers one had 25 and the other 10 members.  

Note: I confirmed with owner of this offering internally and as per Cloud Manager UI maximum allowed user is 30.  I can provide screencapture from Cloud Manager if necessary.  Tnx.

kkour06 committed:
updated microservices urls

kkour06 committed:
Merge pull request #3518 from AdobeDocs/CQDOC-19815-kk

updated microservices urls

guillaumecarlino committed:
Merge branch 'main' into patch-2

guillaumecarlino committed:
Merge pull request #3516 from castaned/patch-2

Update user-access-new-relic.md

singhkh committed:
Forms PDF Utility (#3519)

* Minor edits

* Update image-profiles.md

CQDOC-19753. Added note about CMYK image format not supported.

* Reference Assets for Forms as a Cloud Service-2 (#3420)

* reference-assets

* Update reference-assets.md

* [[DNT - DO NOT TRANSLATE]] Adding ExL tracking IDs to meta data

* fix message

* CQDOC-19806

* fix error

* Reference assets (#3425)

* reference-assets

* Update reference-assets.md

* Minor Change

* Update release-notes-current.md

* Update redirects.csv

* Redirects (#3426)

* reference-assets

* Update reference-assets.md

* Minor Change

* Update release-notes-current.md

* Update redirects.csv

* Updated redirects

* update 1

* [[DNT - DO NOT TRANSLATE]] Adding ExL tracking IDs to meta data

* Update image-profiles.md

CQDOC-19753 - Added "no support for CMYK" in smart crop

* Update limitations.md

CQDOC-19807

* Update release-notes-current.md

* Update image-profiles.md

CQDOC-19807

* Update image-profiles.md

* Additional impl not needed

* CGH-2687-01-01

* CGH-2652-2651-01-01

* CGH-2655-01-01

* Add files via upload

* Update ingesting-content.md

* Update ingesting-content.md

* Rework steps since they were changed and no one told me

* Linking

* Reorg images

* What I could complete of the query module

* Update ToC

* CFM-20221118-01-01

* update 1

* Remove customer name from configuration!

Documentation contains customer name grundfos!
Removing it.

* update 1

* @trivial Add author scale down note

* @trivial Add author scale down note

* @trivial Add author scale down note

* CQDOC-19698

* CQDOC-19696

* Additional steps, update assets path

* Details about the customize app steps

* SKYOPS-43971 - Update dispatcher file structure

* CQ-4347631: add information about test results download
fix HTML comment error

* CQ-4347631: add information about test results download
fix HTML comment error

* fixes according to acrolinx

* @trivial Add reminder to submit ingestion support ticket

* @trivial Add reminder to submit ingestion support ticket

* update 1

* update 2

* update 3

* update 5

* Update screens.md

* Update screens.md

* Rewrite Content Fragment model

* Rearrange content and number steps

* Consistent capitalization

* Rewrite create Content Fragment

* Rewrite extracting content

* Unify structure and cross link

* Update image links

* More cross-linking

* Rewrite first few steps of customize-app

* Finish rewrite of customize-app

* @trivial Add reminder to submit ingestion support ticket
- extra spaces around 'top-up ingestion' might be confusing translations

* @trivial Add reminder to submit ingestion support ticket
- not putting title & abstract on different lines may confuse translations

* Redact beta/staging info from images

* Proofreading

* Applying Acrolinx recommendations that actually make sense

* modified

* Fixed broken image

* minor modifications

* update 1

* Minor changes

* minor changes

* Update help/implementing/developing/introduction/development-guidelines.md

Co-authored-by: Marcel Reutegger <mreutegg@adobe.com>

* Update indexing.md

* Feedback from Alison

* Another catch from Alison

* CQDOC-19737

* Updated previews of apps

* Archive old page and add to ToC

* Copy-paste release notes

* Make release notes readable

* attempt to fix validation

* Added the full document preview support for MSFT files

* Forgot to rename the old release notes file

* Update release-notes-current.md

* [[DNT - DO NOT TRANSLATE]] Adding ExL tracking IDs to meta data

* Clarification of delimiting fields

* Update replication.md

Added the note requested from CQDOC-19839

* Update replication.md

* Add example customer files to file structure as clarification

* Update maintenance.md

Updates for CQDOC-19863

* Add note about configuring the Referrer Filter.

Configuring the Referrer Filter may break AEM's Content Fragment console and Assets Essentials functionality, as the custom Referrer Filter configuration will replace the OOTB configuration relied on by AEM.

* Update caching.md

* Added "Guidelines for defining Smart Crop in an Image profile"

See under "Crop options" heading.

* [[DNT - DO NOT TRANSLATE]] Adding ExL tracking IDs to meta data

* Incorporating PM feedback

* Update image-profiles.md

* CQDOC-19881 fix wrong documentation for asset name sanitization of bulk import

* Update generate-document-of-record-for-non-xfa-based-adaptive-forms.md

* DoR Update

* Update dm-journey-part1.md

Added {target="_blank"} at end of url link to open link into new tab.

* Update limitations.md

* Added syntax to open new tab in "Try it" parts.

* Update dm-journey-part2.md

* Update maintenance.md

* XF-HTMLRG-01-01

* Added enhanced security option for HIPAA

* Pointer to docs on production programs

* Editing enhanced security program

* Updates to metadata and landings for SEO

* PDF Utility

* Minor updates

* Linked to latest API docs

* PDF Utility updates

Fi

Co-authored-by: Anuj Kapoor <anujkapo@adobe.com>
Co-authored-by: Rick Brough <rbrough@adobe.com>
Co-authored-by: Experience League <customer-success@messages.adobe.com>
Co-authored-by: bharat <bwadhwa@adobe.com>
Co-authored-by: Alex Sarchiz <sarchiz@adobe.com>
Co-authored-by: raiman <raiman@adobe.com>
Co-authored-by: Guillaume Carlino <carlino@adobe.com>
Co-authored-by: Patrique Legault <plegault@adobe.com>
Co-authored-by: aheimoz <aheimoz@adobe.com>
Co-authored-by: Aditi Dutt Chaudhuri <duttchau@adobe.com>
Co-authored-by: Chris Bohnert <38424477+bohnertchris@users.noreply.github.com>
Co-authored-by: Bjoern Csott <csott@adobe.com>
Co-authored-by: Andrew Top <top@adobe.com>
Co-authored-by: Florian Froese <ffroese@adobe.com>
Co-authored-by: Ramon Bisswanger <bisswang@adobe.com>
Co-authored-by: Tyler Rushton <trushton@adobe.com>
Co-authored-by: Chris Bohnert <bohnert@adobe.com>
Co-authored-by: Kirandeep Kour <kkour@adobe.com>
Co-authored-by: Marcel Reutegger <mreutegg@adobe.com>
Co-authored-by: David Gonzalez <dgonzale@adobe.com>
Co-authored-by: zjun <zjun@adobe.com>
Co-authored-by: Ruchita <ruchitas@adobe.com>
Co-authored-by: Blake Frei <mfrei@adobe.com>

blefebvre committed:
SITES-10161 - [TBYB] Add guideContextualHelpId entries for "intro" section in learning guide page (#3520)

* SITES-10161 - [TBYB] Add guideContextualHelpId entries for "intro" section in learning guide page

* SITES-10161 - [TBYB] Add guideContextualHelpId entries for "intro" section in learning guide page

Temporarily replace links that were returning 404 during PR validation with a valid image link.

singhkh committed:
Forms CS: Limitations (#3521)

peterklassen committed:
updated the RTO clause again

sarchiz committed:
Merge branch 'main' into top/CQ-4350666-uuid-conflict

peterklassen committed:
RTO updates

guillaumecarlino committed:
Update backup.md

sarchiz committed:
Merge pull request #3515 from AdobeDocs/top/CQ-4350666-uuid-conflict

CQ-4350666 [CAM] Update documentation to explain oak uuid conflicts

guillaumecarlino committed:
Merge branch 'main' into RTO-update

guillaumecarlino committed:
Merge pull request #3522 from AdobeDocs/RTO-update

Rto update

blefebvre committed:
SITES-10161 - [TBYB] Add guideContextualHelpId entries for "intro" section in learning guide page (#3523)

Remove video thumbnail "additional-url" entries and the associated images

Commit:
[[DNT - DO NOT TRANSLATE]] Adding ExL tracking IDs to meta data

Commit:
CQDOC-20114 updating msft scopes

bohnertchris commented:
CQDOC-20114

guillaumecarlino commented:
Closing to align with CQDOC-20114 which was closed.
